User/src/main/java/com/example/user/service/UserService.java

```diff
@@ -0,0 +1,70 @@
+package com.example.user.service;
+
+import com.example.user.Repository.UserRepository;
+import com.example.user.exceptions.ErrorSubmissionException;
+import com.example.user.model.User;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class UserService {
+    private final UserRepository userRepository;
+    public User getUserById(long id){
+        User user=userRepository.existsById(id) ? userRepository.findById(id) : null;
```

Comment:
Nie musisz sprawdzać czy user istnieje, bo repo zwróci ci optionala. Poza tym trochę pokraczne wykorzystanie Optionala - możesz sobie po prostu użyć metody ifPresentOrElse czy coś w tym stylu i będzie to bardziej czytelne